src/components/Search.js

```diff
@@ -0,0 +1,89 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import { Global, css } from '@emotion/core';
+import { DocSearch } from '@docsearch/react';
+
+const Search = () => (
+  <Fragment>
+    <Global
+      styles={css`
+        :root {
+          --docsearch-primary-color: #ff6b81;
+          --docsearch-text-color: rgb(245, 246, 247);
+          --docsearch-container-background: rgba(9, 10, 17, 0.8);
+          --docsearch-modal-background: #281e36;
+          --docsearch-modal-shadow: inset 1px 1px 0 0 rgb(44, 46, 64),
+            0 3px 8px 0 rgb(0, 3, 9);
+          --docsearch-searchbox-background: rgb(9, 10, 17);
+          --docsearch-searchbox-focus-background: #000;
+          --docsearch-hit-color: rgb(190, 195, 201);
+          --docsearch-hit-shadow: none;
+          --docsearch-hit-background: rgb(9, 10, 17);
+          --docsearch-key-gradient: linear-gradient(
+            -26.5deg,
+            rgb(114 86 111) 0%,
+            rgb(65 46 80) 100%
+          );
+          --docsearch-key-shadow: inset 0 -2px 0 0 rgb(81 31 82),
```

Comment:
This property is duplicated line 33.

Reply:
Solved by c60fca5e9ae6a8c522c8596fc9d50cb2340938d2

Reply:
It also resolves the issue raised by the comment about the controls being off-balanced.